lua/yabs/task/init.lua

```diff
@@ -1,4 +1,9 @@
-local Task = {}
+local Task = {
+    scopes = {
+        GLOBAL = 1,
+        LOCAL = 2
```

Comment:
Maybe add `ALL` and assign it automatically to it in all functions that accepts this enum? Just to be more clear instead of just checking for `nil`. Pretty sure that JIT optimize it away.

Reply:
👍 Will do!